2024-05-06 23:44:42,812 INFO    MainThread:9076 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-05-06 23:44:42,812 INFO    MainThread:9076 [wandb_setup.py:_flush():76] Configure stats pid to 9076
2024-05-06 23:44:42,812 INFO    MainThread:9076 [wandb_setup.py:_flush():76] Loading settings from /root/.config/wandb/settings
2024-05-06 23:44:42,812 INFO    MainThread:9076 [wandb_setup.py:_flush():76] Loading settings from /root/mats/sae/wandb/settings
2024-05-06 23:44:42,812 INFO    MainThread:9076 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-06 23:44:42,812 INFO    MainThread:9076 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-06 23:44:42,812 INFO    MainThread:9076 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'sae/main.py', 'program_abspath': '/root/mats/sae/main.py', 'program': '/root/mats/sae/main.py'}
2024-05-06 23:44:42,812 INFO    MainThread:9076 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-06 23:44:42,812 INFO    MainThread:9076 [wandb_init.py:_log_setup():521] Logging user logs to /root/mats/sae/wandb/run-20240506_234442-kdeo6pcm/logs/debug.log
2024-05-06 23:44:42,812 INFO    MainThread:9076 [wandb_init.py:_log_setup():522] Logging internal logs to /root/mats/sae/wandb/run-20240506_234442-kdeo6pcm/logs/debug-internal.log
2024-05-06 23:44:42,812 INFO    MainThread:9076 [wandb_init.py:_jupyter_setup():467] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f30662d9a80>
2024-05-06 23:44:42,812 INFO    MainThread:9076 [wandb_init.py:init():561] calling init triggers
2024-05-06 23:44:42,812 INFO    MainThread:9076 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-05-06 23:44:42,813 INFO    MainThread:9076 [wandb_init.py:init():586] re-initializing run, found existing run on stack: l9zoxk55
2024-05-06 23:44:42,814 INFO    MainThread:9076 [wandb_run.py:_finish():2064] finishing run jacobcd52/sae/l9zoxk55
2024-05-06 23:44:42,814 INFO    MainThread:9076 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2024-05-06 23:44:42,815 INFO    MainThread:9076 [wandb_run.py:_restore():2287] restore
2024-05-06 23:44:42,815 INFO    MainThread:9076 [wandb_run.py:_restore():2293] restore done
2024-05-06 23:44:47,191 INFO    MainThread:9076 [wandb_run.py:_footer_history_summary_info():3936] rendering history
2024-05-06 23:44:47,198 INFO    MainThread:9076 [wandb_run.py:_footer_history_summary_info():3968] rendering summary
2024-05-06 23:44:47,208 INFO    MainThread:9076 [wandb_run.py:_footer_sync_info():3895] logging synced files
2024-05-06 23:44:47,258 INFO    MainThread:9076 [wandb_init.py:init():611] starting backend
2024-05-06 23:44:47,259 INFO    MainThread:9076 [wandb_init.py:init():615] setting up manager
2024-05-06 23:44:47,263 INFO    MainThread:9076 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-05-06 23:44:47,265 INFO    MainThread:9076 [wandb_init.py:init():623] backend started and connected
2024-05-06 23:44:47,274 INFO    MainThread:9076 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2024-05-06 23:44:47,274 INFO    MainThread:9076 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-05-06 23:44:47,274 INFO    MainThread:9076 [wandb_init.py:init():715] updated telemetry
2024-05-06 23:44:47,279 INFO    MainThread:9076 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2024-05-06 23:44:48,532 INFO    MainThread:9076 [wandb_run.py:_on_init():2357] communicating current version
2024-05-06 23:44:48,674 INFO    MainThread:9076 [wandb_run.py:_on_init():2366] got version response 
2024-05-06 23:44:48,675 INFO    MainThread:9076 [wandb_init.py:init():799] starting run threads in backend
2024-05-06 23:44:48,955 INFO    MainThread:9076 [wandb_run.py:_console_start():2335] atexit reg
2024-05-06 23:44:48,955 INFO    MainThread:9076 [wandb_run.py:_redirect():2190] redirect: wrap_raw
2024-05-06 23:44:48,955 INFO    MainThread:9076 [wandb_run.py:_redirect():2255] Wrapping output streams.
2024-05-06 23:44:48,955 INFO    MainThread:9076 [wandb_run.py:_redirect():2280] Redirects installed.
2024-05-06 23:44:48,957 INFO    MainThread:9076 [wandb_init.py:init():842] run started, returning control to user process
2024-05-06 23:46:20,284 INFO    MainThread:9076 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-05-06 23:46:20,284 INFO    MainThread:9076 [wandb_init.py:_pause_backend():432] pausing backend
2024-05-06 23:46:20,368 INFO    MainThread:9076 [wandb_init.py:_resume_backend():437] resuming backend
2024-05-06 23:46:20,378 INFO    MainThread:9076 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-05-06 23:46:20,378 INFO    MainThread:9076 [wandb_init.py:_pause_backend():432] pausing backend
2024-05-06 23:46:24,620 INFO    MainThread:9076 [wandb_init.py:_resume_backend():437] resuming backend
2024-05-06 23:46:24,629 INFO    MainThread:9076 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-05-06 23:46:24,629 INFO    MainThread:9076 [wandb_init.py:_pause_backend():432] pausing backend
2024-05-06 23:46:24,900 INFO    MainThread:9076 [wandb_init.py:_resume_backend():437] resuming backend
2024-05-06 23:46:24,910 INFO    MainThread:9076 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-05-06 23:46:24,910 INFO    MainThread:9076 [wandb_init.py:_pause_backend():432] pausing backend
2024-05-06 23:46:25,192 INFO    MainThread:9076 [wandb_init.py:_resume_backend():437] resuming backend
2024-05-06 23:46:25,194 INFO    MainThread:9076 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-05-06 23:46:25,194 INFO    MainThread:9076 [wandb_init.py:_pause_backend():432] pausing backend
2024-05-06 23:46:25,761 INFO    MainThread:9076 [wandb_init.py:_resume_backend():437] resuming backend
2024-05-06 23:46:25,776 INFO    MainThread:9076 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-05-06 23:46:25,776 INFO    MainThread:9076 [wandb_setup.py:_flush():76] Configure stats pid to 9076
2024-05-06 23:46:25,776 INFO    MainThread:9076 [wandb_setup.py:_flush():76] Loading settings from /root/.config/wandb/settings
2024-05-06 23:46:25,776 INFO    MainThread:9076 [wandb_setup.py:_flush():76] Loading settings from /root/mats/sae/wandb/settings
2024-05-06 23:46:25,776 INFO    MainThread:9076 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-06 23:46:25,777 INFO    MainThread:9076 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-06 23:46:25,777 INFO    MainThread:9076 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'sae/main.py', 'program_abspath': '/root/mats/sae/main.py', 'program': '/root/mats/sae/main.py'}
2024-05-06 23:46:25,777 INFO    MainThread:9076 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-06 23:46:25,777 INFO    MainThread:9076 [wandb_init.py:_log_setup():521] Logging user logs to /root/mats/sae/wandb/run-20240506_234625-za1oi4fm/logs/debug.log
2024-05-06 23:46:25,777 INFO    MainThread:9076 [wandb_init.py:_log_setup():522] Logging internal logs to /root/mats/sae/wandb/run-20240506_234625-za1oi4fm/logs/debug-internal.log
2024-05-06 23:46:25,777 INFO    MainThread:9076 [wandb_init.py:init():561] calling init triggers
2024-05-06 23:46:25,777 INFO    MainThread:9076 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-05-06 23:46:25,777 INFO    MainThread:9076 [wandb_init.py:init():586] re-initializing run, found existing run on stack: kdeo6pcm
2024-05-06 23:46:25,779 INFO    MainThread:9076 [wandb_run.py:_finish():2064] finishing run jacobcd52/sae/kdeo6pcm
2024-05-06 23:46:25,779 INFO    MainThread:9076 [jupyter.py:save_history():445] not saving jupyter history
2024-05-06 23:46:25,780 INFO    MainThread:9076 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-05-06 23:46:25,780 INFO    MainThread:9076 [wandb_init.py:_jupyter_teardown():449] cleaning up jupyter logic
2024-05-06 23:46:25,780 INFO    MainThread:9076 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2024-05-06 23:46:25,780 INFO    MainThread:9076 [wandb_run.py:_restore():2287] restore
2024-05-06 23:46:25,780 INFO    MainThread:9076 [wandb_run.py:_restore():2293] restore done
2024-05-06 23:46:30,679 INFO    MainThread:9076 [wandb_run.py:_footer_history_summary_info():3936] rendering history
2024-05-06 23:46:30,686 INFO    MainThread:9076 [wandb_run.py:_footer_history_summary_info():3968] rendering summary
2024-05-06 23:46:30,695 INFO    MainThread:9076 [wandb_run.py:_footer_sync_info():3895] logging synced files
